contrib/lkt/language/parser.py

```diff
@@ -2285,8 +2285,16 @@ def check_correctness_pre():
 
         return d.result_ref.then(
             lambda d: d.match(
-                # The type ref references a type decl: return no error
-                lambda _=T.TypeDecl: No(T.SemanticResult.array),
+                # The type ref references a type decl: return no error if it
+                # doesn't references a PropertyError.
+                lambda td=T.TypeDecl: If(
+                    td == Entity.property_error_type,
+                    [Entity.error(
+                        S("`").concat(Entity.property_error_type.full_name)
+                        .concat(S("` is not a type"))
```

Comment:
I think we should avoid pretending that `PropertyError` is not a type (it’s a struct in the prelude, so technically it is a type). What about:
> PropertyError can only be referenced in a raise expression